packages/cashier/src/pages/deposit/deposit.tsx

```diff
@@ -1,140 +1,34 @@
-
     React.useEffect(() => {
-        if (!is_crypto_transactions_visible) {
-            recentTransactionOnMount();
-        }
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [is_switching]);
+        setActiveTab('deposit');
+    }, [setActiveTab]);
 
-    React.useEffect(() => {
-        setActiveTab(container);
-        onMount();
-        return () => {
-            setIsDeposit(false);
-            error.setErrorMessage({ code: '', message: '' });
-        };
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [setActiveTab, onMount, container, error.setErrorMessage]);
+    if (is_deposit_locked) return <DepositLocked />;
 
-    React.useEffect(() => {
-        if (typeof setSideNotes === 'function') {
-            if (is_switching || is_deposit) setSideNotes(null);
-            if (is_crypto && is_deposit && !is_switching) {
-                const side_notes = [
-                    ...(crypto_transactions.length ? [<RecentTransaction key={2} />] : []),
-                    ...(/^(UST)$/i.test(currency) ? [<USDTSideNote type='usdt' key={1} />] : []),
-                    ...(/^(eUSDT)$/i.test(currency) ? [<USDTSideNote type='eusdt' key={1} />] : []),
-                ];
-                if (side_notes.length > 0) {
-                    setSideNotes([
-                        <SideNote has_title={false} key={0}>
-                            {side_notes}
-                        </SideNote>,
-                    ]);
-                }
-            }
-        }
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [currency, tab_index, crypto_transactions, crypto_transactions?.length, is_cashier_onboarding, iframe_height]);
-
-    if (!is_cashier_onboarding && (is_switching || (is_loading && !iframe_url)) && !is_crypto_transactions_visible) {
-        return <Loading is_fullscreen />;
-    }
-    if (is_virtual) {
-        return <Virtual />;
-    }
-    if (is_system_maintenance) {
-        if (is_cashier_locked || (is_deposit_locked && current_currency_type === 'crypto')) {
-            return <CashierLocked />;
-        }
-    }
-    if (error.is_ask_uk_funds_protection) {
-        return <FundsProtection />;
-    }
-    if (is_cashier_locked) {
-        return <CashierLocked />;
-    }
-    if (is_deposit_locked) {
-        return <DepositLocked />;
-    }
-    if (is_crypto_transactions_visible) {
-        return <CryptoTransactionsHistory />;
-    }
+    if (is_crypto_transactions_visible) return <CryptoTransactionsHistory />;
 
     if (is_deposit || is_low_risk_cr_eu_real) {
-        if (error.message) {
-            return <Error error={error} />;
-        }
-        if (is_crypto) {
-            return <CryptoDeposit />;
-        }
+        if (is_crypto) return <DepositCryptoModule />;
 
-        return (
-            <>
-                {is_fiat_currency_banner_visible_for_MF_clients && (
```

Comment:
hey hey guys.
@farzin-deriv 
this was implemented because:
when you are CR client and go to /cashier/deposit page -> you will be landed to cashier onboarding page and you can see side note `This is your USD account CR2806546`
but
when you are MF client  and go to /cashier/deposit page -> you will be landed directly to doughflow (iframe) page, and in order to show the client side note we added this flag.
<img width="1780" alt="Screenshot 2023-07-07 at 14 50 22" src="https://github.com/binary-com/deriv-app/assets/103181646/c0a23d9e-518f-4d26-bcc5-305da2eb785e">
![Uploading Screenshot 2023-07-07 at 14.51.39.png…]()

Reply:
@heorhi-deriv Thanks man, Safe to remove then since now each page is responsible for it's own content so these kind of things won't happen anymore 🙇🏻